package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "start": "./bin/server.js",
-    "test": "jshint *.js **/*.js && jscs *.js **/*.js && nodeunit"
+    "test": "jshint {.,bin,lib,lib/repo,test}/*.js && jscs {.,bin,lib,lib/repo,test}/*.js && nodeunit"
```

Comment:
mmm

`jscs .`, whereas in `.jscsrc` config - 

```js
{
  "excludeFiles": [] // to catch `node_modules` folder, since they are in there by default
}
```